test/acceptance-tests/use-cases/test-scripts/find-models.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const app = require('../../../../');
+const expect = require('../../../helpers/expect');
+const fs = require('fs-extra');
+const loopback = require('loopback');
+const ModelClass = require('../../../../component/datamodel/model');
+const path = require('path');
+const testSupport = require('../../../helpers/test-support');
+const util = require('util');
+const workspaceManager = require('../../../../component/workspace-manager');
+
+const ModelDefinition = app.models.ModelDefinition;
+const ModelConfig = app.models.ModelConfig;
```

Comment:
Move these into the declarations inside the module.exports (group with `testsuite`)